stable/kong/templates/deployment.yaml

```diff
@@ -9,8 +9,9 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      {{- include "kong.metaLabels" . | nindent 6 }}
+      app.kubernetes.io/app: {{ template "kong.name" . }}
       app.kubernetes.io/component: app
+      app.kubernetes.io/instance: "{{ .Release.Name }}"
```

Comment:
Should we change the selectors for the Service as well?

Reply:
ah, yea you're right. I just updated all the Services as well.